tests/connection/test_connection_service.py

```diff
@@ -21,9 +21,12 @@
 from ossdbtoolsservice.utils import constants
 from ossdbtoolsservice.utils.cancellation import CancellationToken
 from ossdbtoolsservice.workspace import WorkspaceService
+from ossdbtoolsservice.driver.types.psycopg_driver import PostgreSQLConnection
```

Comment:
Sort import statements alphabetically here and at other places too

Reply:
Would you like me to sort the by sub-modules as well?